spec/brew_dumper_spec.rb

```diff
@@ -477,6 +480,99 @@
     end
   end
 
+  context "when --describe is not set" do
```

Comment:
I wonder if this case is needed just to test `#` isn't added?

Reply:
Eh, it was a TTD test that I left in. It's ensuring that there is no description when the user hasn't requested it. I don't see the harm in leaving it in other than that it extends tests by about 0.7 seconds (`time rspec spec/brew_dumper_spec.rb`).

Reply:
Ok, cool 👍